catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java

```diff
@@ -88,6 +90,11 @@
     public static final String ID = "id";
 
     /**
+     * {@link Attribute} name for accessing the tags of the {@link Metacard}. <br/>
+     */
+    public static final String TAGS = "tags";
```

Comment:
Had a discussion with @harrison-tarr and @michaelmenousek yesterday about how to add attributes that are metadata about the metacard.  We were thinking we should prefix that kind of attribute with `metacard.` to make it clear that the attribute is about the metacard itself and not the product/metadata it represents.  I think tags should become `metacard.tags`.